content/docs/user-guide/scripting/lua/math-library.md

```diff
@@ -0,0 +1,10 @@
+---
+description: Describes the API for common math functions and data types.
+title: Math Library
+toc: true
+weight: 400
+---
```

Comment:
I have another page in the PR that links to this one. Will that link be broken if we mark this as a draft?

Reply:
Good catch, it would be broken. I'm fine with keeping the page in (not setting it as draft), if there's a plan to write this page. I saw an open issue for it - are you assigned to it or know how that issue will move forward?

Reply:
The issue is here https://github.com/o3de/o3de.org/issues/1729 not currently assigned to anyone. I think would be for someone in sig-core to do, not graphics.